CMakeLists.txt

```diff
@@ -381,9 +381,14 @@ add_custom_target(move_artifacts DEPENDS ${stamp_file} ${all_artifacts})
 add_subdirectory (interpreter)
 
 # Update etc/gitinfo.txt for every build.
+find_package(Git)
 add_custom_target(gitinfotxt
```

Comment:
Should there be a:
```
if(Git_FOUND)
   add_custom_target(gitinfotxt
      ....
   )
else()
   message(WARNING "git executable not found, commit info will be inaccurate")
endif()
```
and/or how do we fail if git is missing?

Reply:
Nevermind, `SetROOTVersion.cmake` which is called from `cmake/modules/UpdateGitInfo.cmake` ends up warning if `Git_FOUND` is false.